src/Microsoft.DotNet.Interactive.Formatting/DefaultHtmlFormatterSet.cs

```diff
@@ -155,6 +156,13 @@ internal class DefaultHtmlFormatterSet
                     return formatter.Format(context, value, writer);
                 }),
 
+                // BigInteger should be displayed as plain text
+                new HtmlFormatter<BigInteger>((context, value, writer) =>
+                {
+                    HtmlFormatter.FormatObjectAsPlainText(context, value, writer);
```

Comment:
@jonsequitur you said to write `value.FormatTo(writer, PlainTextFormatter.MimeType);`, but then it would return a number without html wrapping (without `<div class="dni-plaintext">`, `</div>`). So I did it same way as for `object`. What do you think?

Reply:
That works great.